### DISCUSSION TITLE
Onboarding - Button options?

### DISCUSSION THREAD

**Discussion by iespob** [IBM/mac-ibm-notifications#130] [General]
Are there any options to change the button text when using Onboarding? and is it also possible to use a 3rd tertiary button with it?
and thanks for all the awesome work!

**Comment by SMartorelli** [comment]
Hi, currently it's not possible to change the two bottom buttons label, feel free to open a [Feature Request](https://github.com/IBM/mac-ibm-notifications/issues/new?assignees=&labels=&template=feature_request.md&title=%5BFeature+Request%5D).
For the tertiary button there is already a [FR](https://github.com/IBM/mac-ibm-notifications/issues/123) open so you can subscribe to it to receive notification when it will be deployed!

**Reply by iespob** [reply]
Hi Simone, thanks for the quick response! i'll put in an FR.
